lib/subproviders/geth_api_double.js

```diff
@@ -574,6 +574,18 @@ GethApiDouble.prototype.debug_traceTransaction = function(txHash, params, callba
   this.state.queueTransactionTrace(txHash, params, callback);
 };
 
+// Unlocks any account after server start
+GethApiDouble.prototype.debug_unlockAccount = function(address, callback) {
```

Comment:
We chatted about this internally and decided on this method should be named `evm_unlockUnknownAccount`.